src/main/java/com/ensao/gi5/lint/rules/ClassVariableNameRule.java

```diff
@@ -0,0 +1,33 @@
+package com.ensao.gi5.lint.rules;
+
+import com.ensao.gi5.lint.constantes.Constantes;
+import com.ensao.gi5.lint.rules.violations.Violation;
+import com.ensao.gi5.lint.wrapper.CompilationUnitWrapper;
+import com.github.javaparser.ast.body.FieldDeclaration;
+
+public class ClassVariableNameRule extends Rule {
+    public ClassVariableNameRule() {
+        super(Constantes.LINT_REG_004, Level.HIGH);
+    }
+
+    @Override
+    public void apply(CompilationUnitWrapper compilationUnit) {
+        compilationUnit.getCompilationUnit().findAll(FieldDeclaration.class).forEach(fieldDeclaration -> {
+            String name = fieldDeclaration.getVariables().get(0).getNameAsString();
```

Comment:
if a field declaration contains more than one variable, this will not work 
example : 
```java 
private String foo, bar, alice;
```